app/presenters/hyrax/generic_work_presenter.rb

```diff
@@ -1,6 +1,6 @@
 module Hyrax
   class GenericWorkPresenter < Hyrax::WorkShowPresenter
-    delegate :displays_in, to: :solr_document
-    delegate :geographic_name, to: :solr_document
+    include Tufts::SharedTermsService
```

Comment:
I'm not sure I'm following what these `include`s are for, can you help me out? It doesn't look like `SharedTermsService` actually defines any behavior on the presenter or other classes it is included in?